x-pack/plugins/cases/public/components/user_actions/comment/comment.test.tsx

```diff
@@ -482,102 +580,253 @@ describe('createCommentUserActionBuilder', () => {
     expect(screen.getByText('I just isolated the host!')).toBeInTheDocument();
   });
 
-  describe('External references', () => {
+  describe('Attachment framework', () => {
     let appMockRender: AppMockRenderer;
 
     beforeEach(() => {
       appMockRender = createAppMockRenderer();
     });
 
-    it('renders correctly an external reference', async () => {
-      const externalReferenceAttachmentTypeRegistry = new ExternalReferenceAttachmentTypeRegistry();
-      externalReferenceAttachmentTypeRegistry.register(getExternalReferenceAttachment());
-
-      const userAction = getExternalReferenceUserAction();
-      const damagedRaccoon = userProfiles[0];
-      const builder = createCommentUserActionBuilder({
-        ...builderArgs,
-        externalReferenceAttachmentTypeRegistry,
-        caseData: {
-          ...builderArgs.caseData,
-          comments: [
-            {
-              ...externalReferenceAttachment,
-              createdBy: {
-                username: damagedRaccoon.user.username,
-                fullName: damagedRaccoon.user.full_name,
-                email: damagedRaccoon.user.email,
+    describe('External references', () => {
+      it('renders correctly an external reference', async () => {
+        const externalReferenceAttachmentTypeRegistry =
+          new ExternalReferenceAttachmentTypeRegistry();
+        externalReferenceAttachmentTypeRegistry.register(getExternalReferenceAttachment());
+
+        const userAction = getExternalReferenceUserAction();
+        const damagedRaccoon = userProfiles[0];
+        const builder = createCommentUserActionBuilder({
+          ...builderArgs,
+          externalReferenceAttachmentTypeRegistry,
+          caseData: {
+            ...builderArgs.caseData,
+            comments: [
+              {
+                ...externalReferenceAttachment,
+                createdBy: {
+                  username: damagedRaccoon.user.username,
+                  fullName: damagedRaccoon.user.full_name,
+                  email: damagedRaccoon.user.email,
+                },
               },
-            },
-          ],
-        },
-        userAction,
+            ],
+          },
+          userAction,
+        });
+
+        const createdUserAction = builder.build();
+        const result = appMockRender.render(<EuiCommentList comments={createdUserAction} />);
+
+        expect(result.getByTestId('comment-externalReference-.test')).toBeInTheDocument();
+        expect(result.getByTestId('copy-link-external-reference-comment-id')).toBeInTheDocument();
+        expect(result.getByTestId('case-user-profile-avatar-damaged_raccoon')).toBeInTheDocument();
```

Comment:
nit: Here and above `screen` instead of `result` is now the recommended practice